dags/transportation/dockless/dockless_elt.py

```diff
@@ -122,12 +122,16 @@ def load_to_s3(**kwargs):
     config = mds.ConfigFile('/tmp/config.json', company)
     logging.info("Downloaded and parsed config from S3")
     # assert the version parameter
-    version = '0.3.2'
+    try: 
```

Comment:
Not a big deal, but I think this could be done in one line with
```python
getattr(config, 'version', '0.3.2')
```

Reply:
oh, this is much cleaner. Pushing.